mediathread/templates/projects/sequence.html

```diff
@@ -127,12 +167,10 @@
                 <div id="jux-container"></div>
             </div>
             <div role="tabpanel" class="tab-pane fade" id="reflection">
-                {% if project.is_submitted %}
-                    {{project.body|safe}}
-                {% else %}{% if response_can_edit %}
+                {% if response_can_edit %}
```

Comment:
The project body should be visible (but not necessarily editable) to anyone who is viewing the response.